src/main/java/net/pms/configuration/FormatConfiguration.java

```diff
@@ -48,13 +49,14 @@
 	public static final String AIFF = "aiff";
 	public static final String ALAC = "alac";
 	public static final String AMR = "amr";
-	public static final String APE = "ape";
 	public static final String ATRAC = "atrac";
+	public static final String AU = "au";
 	public static final String AVI = "avi";
 	public static final String BMP = "bmp";
 	public static final String CINEPACK = "cvid";
 	public static final String COOK = "cook";
 	public static final String DIVX = "divx";
+	public static final String DSDAudio = "dsd";
```

Comment:
@Sami32 This isn't extensions, this is UMS' "name" for the format. If you check out DSDAudio.java you will find the extensions.